aapp_runner/read_aapp_config.py

```diff
@@ -245,92 +291,129 @@ def check_static_configuration(config):
     return True
 
 
-def read_config_file_options(filename, station, env, valid_config=None):
-    """
-    Read and checks config file
-    If ok, return configuration dictionary
-    """
+def load_config_from_file(filename):
+    """Load the yaml config from file, given the file-path"""
 
     # config = SafeConfigParser()
     import yaml
     with open(filename, 'r') as stream:
         try:
-            config = yaml.load(stream)
+            config = yaml.load(stream, Loader=yaml.FullLoader)
             import pprint
-            print(type(config))
-            pp = pprint.PrettyPrinter(indent=4)
-            pp.pprint(config)
+            # print(type(config))
+            #pp = pprint.PrettyPrinter(indent=4)
+            # pp.pprint(config)
         except yaml.YAMLError as exc:
             print("Failed reading yaml config file: {} with: {}".format(filename, exc))
             raise yaml.YAMLError
 
-    if 'aapp_processes' not in config:
-        print("Can not find main section 'aapp_processes' in yaml file. Please check your config.")
-        return False
+    return config
+
 
-    if valid_config is None:
-        valid_config = VALID_CONFIGURATION
+class AappRunnerConfig(object):
```

Comment:
Is Python 2 still supported?  Should we still subclass from `object` if not?

Reply:
Well I keep the old style. Doesn't really matter. We can make it cutting edge once we have more tests and a more users than the handful I guess we have now ;-)